tests/test_speed.py

```diff
@@ -155,15 +165,14 @@ async def test_class_creation_speed(self) -> None:
             dt = time.monotonic() - t0
             ntopics = len(topic_names)
             creation_speed = ntopics / dt
-            print(
+            self.log.debug(
```

Comment:
This message is important data. It must be info level. That is true of all the other messages that contain measured results.